docs/metadataPackage.md

```diff
@@ -31,15 +31,16 @@ flowchart LR;
 
 ### ArchiveFolder
 
-| Name        | Required | Description                                                                                                                                                                                                           |
-| ----------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| description | no       | The description of the folder. On ingest the description of the folder in the Preservation System is set to this value if present, on subsequent ingests the description is only updated if a `description` is given. |
-| id          | yes      | A unique identifier within the ingest package for this folder. As this is the ArchiveFolder type it will be stored in the Preservation System and used as the key for future updates to this folder.                  |
-| id\_\*      | no       | Sets an identifier in the Preservation System. E.g. `id_Code` becomes the `Code` identifier.                                                                                                                          |
-| parentId    | yes\*    | The `id` of the object's parent (also within this JSON file).                                                                                                                                                         |
-| series      | yes\*    | If the item is a root node in this ingest package set this to the series reference the item should ingest under or `Unknown`.                                                                                         |
-| title       | no       | The title of the folder. On ingest the title of the folder in the Preservation System is set to this value if present, on subsequent ingests the title is only updated if a `title` is given.                         |
-| type        | yes      | The item type (`ArchiveFolder`).                                                                                                                                                                                      |
+| Name        | Required | Description                                                                                                                                                                                                                     |
+| ----------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| description | no       | The description of the folder. On ingest the description of the folder in the Preservation System is set to this value if present, on subsequent ingests the description is only updated if a `description` is given.           |
```

Comment:
1. There's a comma missing after "On ingest"
2. I think "if present, on subsequent ingests the" should be "if present; on subsequent ingests, the"